R/dbWriteTable.R

```diff
@@ -42,7 +42,7 @@
 #' # No row names
 #' dbWriteTable(con, "mtcars", mtcars[1:10, ], overwrite = TRUE, row.names = FALSE)
 #' dbReadTable(con, "mtcars")
-#' @export
```

Comment:
Is this intended?

Reply:
There were multiple (2) `@export` tags in there.